regress/regress_expected

```diff
@@ -87,7 +87,7 @@ ERROR:  parse error - invalid geometry at character 23
 76|f
 76a|f
 106|BOX3D(0 0 0,7 7 0)
-107|POLYGON((0 0,0 7,7 7,7 0,0 0))
+107|POLYHEDRALSURFACE(((0 0 0,0 7 0,7 7 0,7 0 0,0 0 0)),((0 0 7,0 7 7,7 7 7,7 0 7,0 0 7)),((0 0 0,0 0 7,0 7 7,0 7 0,0 0 0)),((7 0 0,7 0 7,7 7 7,7 7 0,7 0 0)),((0 0 0,0 0 7,7 0 7,7 0 0,0 0 0)),((0 7 0,0 7 7,7 7 7,7 7 0,0 7 0)))
```

Comment:
Point and line would be good too, thought I was thinking of just the three different code paths for generating a polygon (flat in z plane, flat in y plane, flat in x plane)